aws/grafana-backing-services/Makefile

```diff
@@ -0,0 +1,15 @@
+## Initialize terraform remote state
+init:
+	[ -f .terraform/terraform.tfstate ] || terraform $@
+
+## Clean up the project
+clean:
+	rm -rf .terraform *.tfstate* $$TF_MODULE_CACHE
```

Comment:
```suggestion
	rm -rf .terraform *.tfstate* "${TF_MODULE_CACHE:-.terraform}"
```

Reply:
I don't know why I proposed "${TF_MODULE_CACHE:-.terraform}" and I don't think it works anyway. `$$TF_MODULE_CACHE` works and is nicer because if `TF_MODULE_CACHE` is not set, then it simply has no effect.